From 9c6fd3bba91930deaf7d86f4af1739999e805934 Mon Sep 17 00:00:00 2001
From: Jorge Ramirez-Ortiz <jorge@foundries.io>
Date: Wed, 20 Nov 2024 17:43:27 +0100
Subject: [PATCH] regression 1000: disable 1039

This test requires the TA to be signed with a key located
in the optee_test repository.

We can not do that with LmP currently - and it is probably not
worth the effort at this time

Upstream-Status: Inappropriate [lmp specific]

Signed-off-by: Jorge Ramirez-Ortiz <jorge@foundries.io>
---
 host/xtest/regression_1000.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/host/xtest/regression_1000.c b/host/xtest/regression_1000.c
index 9981d01..226e5dc 100644
--- a/host/xtest/regression_1000.c
+++ b/host/xtest/regression_1000.c
@@ -3269,6 +3269,7 @@ out:
 ADBG_CASE_DEFINE(regression, 1038, xtest_tee_test_1038,
 		 "Test MTE (Memory Tag Extension)");
 
+#if 0
 static void xtest_tee_test_1039(ADBG_Case_t *c)
 {
 	TEEC_Session session = { };
@@ -3291,6 +3292,7 @@ static void xtest_tee_test_1039(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(regression, 1039, xtest_tee_test_1039,
 		 "Test subkey verification");
+#endif
 
 struct test_1040_thread_arg {
 	TEEC_Result res;
-- 
2.34.1

